benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -227,6 +227,14 @@ msgctxt "image alt text"
 msgid "core.icons.bankcard"
 msgstr "Bank card icon with contactless symbol"
 
+#: benefits/eligibility/forms.py:23
+msgid "eligibility.form.continue"
+msgstr "Continue"
+
+#: benefits/eligibility/forms.py:24
+msgid "eligibility.form.continuing"
+msgstr "Continuing"
+
```

Comment:
Referring to the [Figma file for this page](https://www.figma.com/file/5IabSdY0xDzwL4gyXhbQ9G/Benefits-(Courtesy-Cards)?node-id=1618%3A4627), there is no second-state for this button. (It's a simple form anyway)